### PR TITLE
Fe home page create

### PR DESCRIPTION
**Pull request by jshortz**
Home page created, uses Milad's color palette.

### CODE REVIEW

File: frontend/src/components/Home/Home.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    <div id="home-header">
+                        A unique, easy-to-use project management tool
+                    </div>
+                    <div id="home-blurb">
+                        PerFlow is a new project management tool. At first glance, it seamlessly provides
+                         a number of visual representations of how your project is going. Behind the scenes, algorithms
+                         calculate data that a project management team has traditionally had to laboriously calculate by hand.
+                    </div>
+                    <div>
+                        <a href="/Register" style={{textDecoration: 'none'}}><button type="button" class="btn btn-primary btn-lg" id="home-signup-button">Sign Up For A Free Trial</button></a>
```
Comment (pratima-shegokar) [inline frontend/src/components/Home/Home.js]: The registration url is /RegistrationPage. Can you please update this pr with the corresponding change?

File: frontend/src/App.css [hunk 2]
```diff
@@ -21,7 +21,30 @@
...
+
+#home-signup-button {
+    margin: 2em auto 0px auto;
+    width: 14em;
+    height: 3em;
+    display: block;
+    font-size: 150%;
+    background-color: #84CEEB;
+}
+
```
Comment (MelanyCristy) [inline frontend/src/App.css:56]: Why did you use this new css inside app.css?

Comment (jshortz) [reply frontend/src/App.css:56]: Do you mean as opposed to doing it inline?

**Review by MiladYaqo — APPROVED** [review]
Looks good for now, the css files need to be in a separate file.

**Pull request merged**